src/Application/Clans/Models/ClanMemberViewModel.cs

```diff
@@ -8,4 +8,5 @@ public record ClanMemberViewModel : IMapFrom<ClanMember>
 {
     public UserPublicViewModel User { get; init; } = default!;
     public ClanMemberRole Role { get; init; }
+    public Clan? Clan { get; set; }

```

Comment:
The web api shouldn't expose any domain object. Create a view for the object so you can filter out any unwanted fields.

Reply:
I think you can use the object returns by /users/self/clans

Reply:
This was what i planned , but for some reason i forgot -.-"